NSS-Therapeutic-Raccoons/WorkforceManagement
# Rb trainingprogram listcreate

## Link to Ticket
Closes # [9](https://github.com/NSS-Therapeutic-Raccoons/WorkforceManagement/issues/9)

## Description of Proposed Changes
- Created mvc for Training Programs
- Created integration test for Training Programs
- Created Index, Create MVC in `TrainingProgramController`
- Created Index and Create IntegrationTest in `TrainingProgramTests`
- Created Directory for TrainingProgram Views
- Created Index and Create View in `Views/TrainingProgram`

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout rb-trainingprogram-listcreate
```
run the tests and verify in browser
examine various new files to ensure no problems


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Nothing already build, aside from an added navbar and home page link to the training programs.
* `BangazonWorkforce/Controllers/TrainingProgramController.cs` added
* `BangazonWorkforce/Models/ViewModels/TrainingProgramIndexViewModel.cs` added
* `BangazonWorkforce/Models/ViewModels/TrainingProgramsCreateViewModel.cs` added
* `BangazonWorkforce/Views/TrainingProgram/Create.cs` added
* `BangazonWorkforce/Views/TrainingProgram/Index.cs` added
* `BangazonWorkforce.IntegrationTests.TrainingProgramTests.cs` added

## Mentions @username

@khardt
@figamus
@jeremiah3643
@ricky-bruner
@askingalot

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
